src/lib/PackageList.svelte

```diff
@@ -4,7 +4,7 @@
   import PackageItem from "$lib/PackageItem.svelte";
 </script>
 
-<div class="results_count">We found: {$results_count} Plone add-on's</div>
+<div class="results_count">We found: {$results_count} Plone packages</div>
```

Comment:
I'm sure we should change the naming here. So far in the Plone world we always called them add-on and not package. Hence the name of the project.

Reply:
The terms are not used consistently. In the UI you refer to packages in two different places, and add-on is a type of package.

![Screenshot 2024-10-25 at 2 57 58 AM](https://github.com/user-attachments/assets/a7e18c7e-a66f-453a-9daf-ae72d932fa1e)

Also I think "Plone Add-on Gallery" is misnamed and should be "Plone Package Gallery", based on what it actually searches for.

Reply:
Then i think i rather change the Package Types in to "Addon Types", as everywhere in Plone we call them addons.
I know the Python world calls them differently but that is a different context. In Plone extensions are called aadons, even in Volto that is the case.

Reply:
Well, npm calls them "packages", so it's not just Python.

I'll replace "package" with "add-on", though, just for consistency. Inconsistency bugs me more than exact name.

Reply:
@MrTango I think I got it now. Please check. Thank you!